api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -374,11 +374,30 @@ type NodeAgentCommonFields struct {
 	PodConfig *PodConfig `json:"podConfig,omitempty"`
 }
 
-type RestorePVC struct {
-	// IgnoreDelayBinding indicates to ignore delay binding the restorePVC when it is in WaitForFirstConsumer mode
-	IgnoreDelayBinding bool `json:"ignoreDelayBinding,omitempty"`
+// LoadConcurrency is the config for data path load concurrency per node.
+// Struct should be same as:
+// https://github.com/openshift/velero/blob/584cf1148a746838ee67aa27e3e4e0ded1f5c069/pkg/nodeagent/node_agent.go#L52-L58
+type LoadConcurrency struct {
+	// GlobalConfig specifies the concurrency number to all nodes for which per-node config is not specified
+	GlobalConfig int `json:"globalConfig,omitempty"`
+
+	// PerNodeConfig specifies the concurrency number to nodes matched by rules
+	PerNodeConfig []RuledConfigs `json:"perNodeConfig,omitempty"`
 }
 
+// LoadAffinity is the config for data path load affinity.
+// Used by the Node-Agent, that needs to match the DataMover and the RepositoryMaintenance pods.
+// Struct should be same as:
+// https://github.com/openshift/velero/blob/584cf1148a746838ee67aa27e3e4e0ded1f5c069/pkg/nodeagent/node_agent.go#L60-L63
+type LoadAffinity struct {
+	// NodeSelector specifies the label selector to match nodes
+	// +optional
```

Comment:
this is not optional in velero side. Can this be a problem?

Reply:
In which Velero CRD are you seeing this as exposed non-optional filed?

Reply:
not CRD (this is used only to read configMap, right?), but on velero struct it does not have optional marker or omitempty json tag